packages/runtime/src/widgets/Form/TextInput.tsx

```diff
@@ -80,6 +81,17 @@ export const TextInput: React.FC<TextInputProps> = (props) => {
     target.value = target.value.replace(/[^0-9.]/g, "");
   }, []);
 
+  const handleInputPaste = useCallback(
+    (e: React.ClipboardEvent) => {
+      const pastedData = e.clipboardData.getData("Text");
```

Comment:
This should be lower case `text`? According to the standard https://html.spec.whatwg.org/multipage/dnd.html#dom-datatransfer-getdata